yandex/mdb_mongodb_structures.go

```diff
@@ -110,12 +110,19 @@ func parseMongoDBEnv(e string) (mongodb.Cluster_Environment, error) {
 }
 
 func mongodbUserPermissionHash(v interface{}) int {
+	var buf bytes.Buffer
+
 	m := v.(map[string]interface{})
 
 	if n, ok := m["database_name"]; ok {
-		return hashcode.String(n.(string))
+		buf.WriteString(fmt.Sprintf("%s-", n.(string)))
```

Comment:
Changing the hash function will cause 'terraform diff` to appear when switching to a new version of the provider. Migration should be written.

Reply:
Oh, you're probably right.
I'll try to implement that and add a commit to the PR.